docker/kubernetes-agent-tentacle/scripts/configure-and-run.sh

```diff
@@ -47,6 +47,28 @@ function getPublicHostName() {
 }
 
 function validateVariables() {
+  validateCommonVariables
+
+  if [[ "$DeploymentTargetEnabled" != "true" && "$WorkerEnabled" != "true" ]]; then
```

Comment:
For my understanding: Is it that neither of these existed before, and now these do exist. Could the requirement of these variables break a customer or is it that the helm chart always takes care of setting one of these and so no customer can be impacted?

Reply:
Good question. We are planning to move to a new [v2 Helm chart](https://github.com/OctopusDeploy/helm-charts/pull/196) which will be setting these env variables. We will also be [updating](https://github.com/OctopusDeploy/helm-charts/pull/220) the current Helm chart (v1) to set these variables so that it can continue using the latest Tentacle versions.